src/controllers/connection/newConnection.ts

```diff
@@ -121,7 +124,7 @@ export class NewConnectionController extends HTMLController {
 
     const companyOrError = await this.lookupCompany(req.log, companyNumber, registryCountryCode)
     if (companyOrError.type === 'error') {
-      req.log.warn('Error occured while looking up the company %j', {
+      req.log.warn('Error occurred while looking up the company %j', {
```

Comment:
did you merge main? I thought I corrected all spellings in one of my previous PRs...